docs/index.md

```diff
@@ -1,17 +1,27 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: InternBuddy
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S2-CS2103T-W14-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103T-W14-3/tp/actions)
 [![codecov](https://codecov.io/gh/CS2103T-W14-3/tp/branch/master/graph/badge.svg?token=IHVW9KCP82)](https://codecov.io/gh/CS2103T-W14-3/tp)
```

Comment:
Could you also change this line to this:https://codecov.io/gh/AY2122S2-CS2103T-W14-3/tp/branch/master/graph/badge.svg?token=IHVW9KCP82

Reply:
Yes, no problem